Doc/library/http.server.rst

```diff
@@ -55,131 +55,131 @@ provides three different variants:
 
 .. class:: BaseHTTPRequestHandler(request, client_address, server)
 
-   This class is used to handle the HTTP requests that arrive at the server.  By
+   This class is used to handle the HTTP requests that arrive at the server. By
    itself, it cannot respond to any actual HTTP requests; it must be subclassed
    to handle each request method (e.g. GET or POST).
    :class:`BaseHTTPRequestHandler` provides a number of class and instance
    variables, and methods for use by subclasses.
 
    The handler will parse the request and the headers, then call a method
    specific to the request type. The method name is constructed from the
-   request. For example, for the request method ``SPAM``, the :meth:`do_SPAM`
+   request. For example, for the request method SPAM, the :meth:`do_SPAM`
    method will be called with no arguments. All of the relevant information is
-   stored in instance variables of the handler.  Subclasses should not need to
+   stored in instance variables of the handler. Subclasses should not need to
    override or extend the :meth:`__init__` method.
 
-   :class:`BaseHTTPRequestHandler` has the following instance variables:
+   :class:`BaseHTTPRequestHandler` defines the following instance variables:
```

Comment:
Are you disturbed by the wording that a class «has» isntance variables?
I’m not sure that people would not understand what is meant here.  But it’s also a straightforward change, so why not.